### PR TITLE
OO-5867 update y axis domain

### PR DESCRIPTION
**Pull request by findo11**

### CODE REVIEW

**Review by stepanstava — CHANGES_REQUESTED** [review]

File: packages/dashboards/src/lib/components/timeseries-widget/timeseries-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    );
+
+    // case where there are no measurements for non-percent metrics,  e.g when a license expires
+    if (nonPercentMetrics.length === 0) {
+        return { min: -1, max: 1 };
+    }
+
+    const measurements = nonPercentMetrics
+        .map((m) => m.data)
+        .reduce((x, y) => x.concat(y), []);
```
Comment (stepanstava) [inline packages/dashboards/src/lib/components/timeseries-widget/timeseries-helpers.ts:44]: isn't this flatMap()?
https://github.com/solarwinds/orion/blob/main/src/frontend/fusion/packages/platform-sdk/src/dashboard/datasources/timeseries/perfstack-metrics-converter/perfstack-metrics-utils.ts#L49

File: packages/dashboards/src/lib/components/timeseries-widget/chart-presets/timeseries-chart.component.ts [hunk 2]
```diff
@@ -203,6 +210,28 @@ export abstract class TimeseriesChartComponent<T = ITimeseriesWidgetSeriesData>
...
+    public updateYAxisDomain(): void {
+        const scaleKeys = ["y", "yRight"] as Array<
+            keyof ITimeseriesScalesConfig
+        >;
+        for (const scaleKey of scaleKeys) {
+            const scaleConfig = this.configuration.scales?.[scaleKey];
+            if (scaleConfig?.properties) {
+                scaleConfig.properties.domain = {
+                    ...metricsSeriesMeasurementsMinMax(
+                        this.widgetData.series as any,
```
Comment (jirkapok) [inline packages/dashboards/src/lib/components/timeseries-widget/chart-presets/timeseries-chart.component.ts:222]: Are we able to avoid any?

**Review by jirkapok — APPROVED** [review]

File: packages/dashboards/src/lib/components/timeseries-widget/chart-presets/timeseries-chart.component.ts (hunk 2 above)
Comment (stepanstava) [inline packages/dashboards/src/lib/components/timeseries-widget/chart-presets/timeseries-chart.component.ts:222]: why any here?

**Review by stepanstava — APPROVED** [review]

**Review by stepanstava — CHANGES_REQUESTED** [review]
percy dashboards tests:
Before:
image

After:
image

**Review by stepanstava — APPROVED** [review]

**Pull request merged**